### DISCUSSION TITLE
How do I remove the 'field type' in Query results header??

### DISCUSSION THREAD

**Discussion by kf6spf** [Sequel-Ace/Sequel-Ace#1766] [Q&A]
So I used to use Sequel Pro and recently discovered that it was "gone" and came across this program.

Very nice!!

Only question is how do I remove the column type listings in the header of Query Results??

![SequelAceQueryColumnType](https://github.com/Sequel-Ace/Sequel-Ace/assets/18036760/29bfbace-eb2b-4917-a75d-f80eabf3238e)

**Comment by Kaspik** [comment]
It's in the settings I believe :)

**Reply by samuelbradshaw** [reply]
The "Display column types on content view" option in Settings only applies to table headers in the Content tab, unfortunately – it doesn't affect table headers in the Query tab. There isn't a way that I've found to turn the column types off in the Query tab.

**Reply by kf6spf** [reply]
Sigh ... that's my conclusion too. Guess this becomes an Enhancement Request.

**Reply by Jason-Morcos** [reply]
Just a bug, anyone is welcome to submit a PR to address this and I'd be happy to review and merge :)
